src/Routing/RouteCollectionBuilder.php

```diff
@@ -13,159 +13,203 @@
 
 namespace MonsieurBiz\SyliusNoCommercePlugin\Routing;
 
+use MonsieurBiz\SyliusNoCommercePlugin\Model\ConfigInterface;
 use Symfony\Component\Config\Exception\LoaderLoadException;
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\Config\Resource\ResourceInterface;
-use Symfony\Component\Routing\Route;
 use Symfony\Component\Routing\RouteCollection;
 use Symfony\Component\Routing\RouteCollectionBuilder as BaseRouteCollectionBuilder;
 
 class RouteCollectionBuilder extends BaseRouteCollectionBuilder
 {
     private ?LoaderInterface $loader;
+    private ConfigInterface $config;
     private array $resources = [];
 
     private array $routesToRemove = [
         // Customers & Account & Users
-        'sylius_admin_partial_customer',
-        'sylius_admin_customer',
-        'api_customer',
-        'sylius_shop_log',
-        'sylius_shop_register',
-        'sylius_shop_request_password_reset_token',
-        'sylius_shop_password_reset',
-        'sylius_shop_user_request_verification_token',
-        'sylius_shop_user_verification',
-        'sylius_shop_account',
-        'api_register_shop_users_post_collection',
-        'sylius_api_shop_authentication_token',
-        'sylius_shop_ajax_user_check_action',
+        'customer' => [
```

Comment:
Could these values not be in a config file?

Reply:
Later yes :)